### PR TITLE
bump default gRPC send max size to 100Mb

### PR DESCRIPTION
**Pull request by ortuman**
**What this PR does**:

Bump `grpcclient` default max send size from 16Mb to 100Mb.

**Which issue(s) this PR fixes**:

Fixes N/A

**Checklist**
- [ ] Tests updated
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by pracucci — COMMENTED** [review]
I agree on keeping it consistent with `grpc-max-recv-msg-size`, but before merging we should get a review from Loki / Tempo too.

File: CHANGELOG.md
Comment (replay) [inline CHANGELOG.md:21]: Is the change on this line accidental?

Comment (pracucci) [reply CHANGELOG.md]: I take care of fixing this, so we can move forward with this PR.

**Review by pracucci — APPROVED** [review]

**Pull request merged**
